inc/styles.php

```diff
@@ -526,7 +526,8 @@ static function filter_css_object( $css, $panels_data, $post_id, $layout ) {
 			}
 
 			// Add in flexbox alignment to the main row element
-			if ( ! empty( $row['style']['cell_alignment'] ) ) {
+			$flexbox_align_styles = array('auto','flex-start', 'center', 'flex-end', 'stretch');
+			if ( ! empty( $row['style']['cell_alignment'] ) && in_array($row['style']['cell_alignment'], $flexbox_align_styles ) ) {
```

Comment:
I don't think we should be hard-coding the options here. Then we have to remember to add new options here and in the `row_style_fields` function. If you're not using `flexbox` then perhaps you can filter out the `cell_alignment` field entirely and add a custom `nofb_cell_alignment` field.